google/protobuf/empty_pb2.py

```diff
@@ -19,8 +17,8 @@
   name='google/protobuf/empty.proto',
   package='google.protobuf',
   syntax='proto3',
-  serialized_options=_b('\n\023com.google.protobufB\nEmptyProtoP\001Z\'github.com/golang/protobuf/ptypes/empty\370\001\001\242\002\003GPB\252\002\036Google.Protobuf.WellKnownTypes'),
-  serialized_pb=_b('\n\x1bgoogle/protobuf/empty.proto\x12\x0fgoogle.protobuf\"\x07\n\x05\x45mptyBv\n\x13\x63om.google.protobufB\nEmptyProtoP\x01Z\'github.com/golang/protobuf/ptypes/empty\xf8\x01\x01\xa2\x02\x03GPB\xaa\x02\x1eGoogle.Protobuf.WellKnownTypesb\x06proto3')
+  serialized_options=b'\n\023com.google.protobufB\nEmptyProtoP\001Z\'github.com/golang/protobuf/ptypes/empty\370\001\001\242\002\003GPB\252\002\036Google.Protobuf.WellKnownTypes',
```

Comment:
Looks like there are still weird formatting changes happening somehow. Maybe related to `buf` versions?

Reply:
Hum, I thought we had settled on a similar version. I'm at 0.8.0 right now.
Should we choose a version for everyone to use?

Reply:
We could probably address this by moving all the stub building into GH actions. We were on 0.7.1 last I knew, but no objections to upgrading.

Reply:
I agree on moving the build to GH actions.
The only negative thing I can see about that is that building locally was useful for testing. But we can still do that and only commit .proto files

Reply:
Even if the other files get committed, I think the action will override them with whatever it thinks the official version should look like. 🤷‍♂